Earthfile

```diff
@@ -477,6 +491,39 @@ arm-image:
   END
   SAVE ARTIFACT /build/$IMAGE_NAME.sha256 img-sha256 AS LOCAL build/$IMAGE_NAME.sha256
 
+prepare-arm-image:
```

Comment:
cant we reuse the existing arm image stuff? What is the difference from other arm images?

Reply:
we don't actually create an SD card here, we just prepare the partitions